File: TestTask.UI/src/app/models/response/GetLogsResponse.ts
import { Log } from "../Log";
import { BaseResponse } from "./BaseResponse";

export class GetLogsResponse extends BaseResponse{
    data : Log[] = [];
    count : number = 0 ;
    pageNumber : number = 0;
    pageSize : number = 0;
    totalCount : number = 0;
}

File: TestTask.BLL/Response/GetLogsResponse.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TestTask.BLL.DTO;

namespace TestTask.BLL.Response
{
	public class GetLogsResponse : BaseResponse
	{
		public List<LogDTO> Data { get; set; }
		public int Count { get; set; }
		public int PageNumber { get; set; }
		public int PageSize { get; set; }
		public int TotalCount { get; set; }
	}
}
